themes/digital.gov/src/scss/new/shortcodes/_highlight.scss

```diff
@@ -1,7 +1,12 @@
 @use "uswds-core" as *;
 
 .highlight {
-  @include u-padding-y("1px");
-  @include u-padding-x("05");
-  @include u-bg("yellow-10v");
+  @include u-padding-y(1);
+  @include u-padding-x(2);
+  @include u-bg("blue-warm-5");
```

Comment:
Can we get rid of these so it aligns with the content in boxes?

```suggestion
```

**Example**
![image](https://github.com/GSA/digitalgov.gov/assets/3385219/a87a187a-f54e-42f2-93ac-9164e63060d3)

Reply:
I see what you are saying... are we ok with it looking like this outside of the box shortcode?

![Screenshot 2024-04-23 at 1 07 24 PM](https://github.com/GSA/digitalgov.gov/assets/88721460/d27a698e-0034-4cfc-9541-408057234d0d)

Reply:
are we ok with it looking like this outside of the box shortcode?

![Screenshot 2024-04-23 at 1 07 24 PM](https://github.com/GSA/digitalgov.gov/assets/88721460/88d0ac1e-3e35-4dc0-8f36-fab9d56764e3)

Reply:
Yes, that's fine for now to clearly highlight the code block.